### DISCUSSION TITLE
July Monthly Challenge

### DISCUSSION THREAD

**Discussion by BekahHW** [Virtual-Coffee/VC-Community-Docs#155] [Monthly Challenges]
@Virtual-Coffee/monthly-challenge 

Let's look ahead and brainstorm next month! How can we build on daily standup and work on some focused goals and projects?

**Comment by BekahHW** [comment]
I know we're off the [schedule that @tkshill suggested](https://github.com/Virtual-Coffee/VC-Contributors/discussions/144#discussioncomment-782718), but I think we can adjust as we go. Maybe kind of switch here. This is kind of month of sharing. Kicking off standup. What if next month is building? We don't have to end with an event, but we _can_ do DemoDays or as an alternative people can do a video walk through, or blog. 

And maybe this is where the _challenge_ part of things comes into play. Our theme is Focus on one thing or something like that. And then the challenge is to present your progress--it doesn't have to be done--at Demo Days or in another format.

What are some other ways that could work to build on standup?

**Reply by clandau** [reply]
I like the idea of demos, maybe once or twice during the month you're expected to demo. and have multiple options / days where you could demo.

**Reply by vanessacor** [reply]
I like this idea, having a demo days each week where people can sign-up to it

**Comment by Andrew-Bush** [comment]
Well stand-up is part of an agile workflow.  So, some natural extensions could be the different associated ceremonies.  
- Sprint planning
- Retrospective
- Backlog grooming / Story time

#incompleteThought

**Comment by rhdeck** [comment]
Could we simplify and create durable media assets for the participants at
the same time? Month of demos. Goal is to dial in the skill of teaching how
to use a project/product via async video. Could be your own project or that
of someone else that you are providing training on.

We open with a "how to demo with loom" at the start of the month. Everyone
is encouraged to ship a "shitty first draft" by the end of the first week.
The expectation is supportive community review of each others' demos and
then ship a new draft the following week. For next three weeks a different
draft of the demo gets made with Loom (or tella, vidyard etc) and posted to
#monthly-challenge. Four weekly cycles with the goal (like all of this,
optional)  to post on YT at the end?

**Reply by BekahHW** [reply]
@rhdeck I'm interested in thinking about this along with what @vanessacor and @clandau said above. I want to make this open to the widest group of VC-ers as possible. So sometimes demo-ing a personal project just isn't doable. So I definitely like your idea "Could be your own project or that of someone else that you are providing training on."

Even with VC, I've just started making looms to navigate our processes here. So potentially members can contribute to that, and we have a number of OSS member projects that could buy-in.

I wonder if that's too much though. So I'd love to hear other thoughts.

**Comment by LSanchez17** [comment]
I like the idea of next month being building.  It would be building (pun not intentional) upon maybe a topic, project, or technology somebody learned during the learning in public month.  

I'm not 100% sure how feasible it would be for most people to create a durable project out of it.  Should we have a rough outline, in case any new members aren't well versed in deployment or creating content out of their learning/projects?  How would we adjust if someone wants to join during the middle of the month?

The standups and daily slack bot reminders are a nice touch.  Does slack offer any sort of statistical or analytics per channel? We could use that to gauge how each challenge/month pans out in terms of posts, reactions, etc.

Just some thoughts I've been having 😊

**Reply by BekahHW** [reply]
@LSanchez17 Sorry it's taken so long to respond to this. I like the idea of tracking slack for engagement. Our challenges have been all over the place in terms of where and how they happen. So we did pairing, and most of that was done via zoom and PRs. Blogposts was mostly done on personal blogs and via PRs. 

But if we can figure out a way to do this differently or to include the slack component, I'm all for that.

**Comment by adam-abundis** [comment]
What if we do a "weekly" project and give the teams a theme rather than a specific project? You could change the teams every week. At least with a theme, the groups have more creative control in creating a project that interests them. And it gives everyone 4 chances to participate for the month.

**Reply by BekahHW** [reply]
My concern here is that we're not ready to support teams yet. Getting schedules together is really challenging and with ppl at all different levels and timezones, the logistics can get hard to coordinate.

I am definitely interested in the idea of a theme. And I would be more inclined to say something like here's a theme of X. We have two tracks, one is individual and one is for a team. For a team, the goal is to meet three times to work on your theme and present, either async or sync on the last week your progress. Or for individuals to do 3 check-ins (sync/async) and the fourth week present in the same way.

It doesn't have to be a finished project, but something that you've made progress on. And maybe the theme is just an optional part, idk. I can see major progress being made by setting up a repo, creating documentation, adding issues, and having no code. But the next goal would be to add code and work from there.

@rhdeck I wonder if you have thoughts based on the cohort style groups you've been running.
